docs/Run_Iml_Tests_Cygwin.md

```diff
@@ -0,0 +1,89 @@
+[**Table of Contents**](index.md)
+
+# Running IML tests on HPC cluster with Windows, Cygwin and Vagrant
```

Comment:
This really should be broken down into a major section about running SSI Tests with sub-sections about the differences between doing it on Windows, Linux and MacOS.  Windows probably ought not be the first subsection.  I would guess our audience is going to be people very comfortable with Linux and/or MacOS primarily given that IML is all about Linux.

Reply:
Agreed. Making note.

Reply:
In the interest of time, can we open an issue to describe this cross OS and proceed with this version?

Reply:
I'm really quite sceptical that the majority of contributors wanting to run SSI tests are going to be (only) using Windows machines, so the fact that our only documentation on running SSI tests is geared towards Windows users seems quite problematic.

Reply:
Brian, I sent a you a skype message and requested that you add instructions for Linux and possibly MacOS.